### DISCUSSION TITLE
Unity 월드 제작시 에셋스토어 사용 범위

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#539] [5. Q&A]
![image](https://user-images.githubusercontent.com/50188795/160515305-01b61784-e0ef-47e8-9d14-970cb51f0baf.png)

유니티로 월드 제작시 에셋스토어에서 구매한 에셋은 사용가능한가요?

**Comment by x4cf** [comment]
라이센스상에는 문제가 없지만 C#으로 작성된 코드는 사용할 수 없습니다.

**Reply by smilejsu82** [reply]
스크립트가 포함안된 모델들만 사용하려고 합니다 

그리고 리소스에 포함된 쉐이더 파일들은 그대로 사용할수 있나요?

**Reply by x4cf** [reply]
사용가능하며 저도 사용중입니다.
*몇몇 쉐이더 파일들은 작동못해요 :)

**Reply by smilejsu82** [reply]
커스텀 쉐이더도 작성해서 사용가능한가요? 
몇몇 쉐이더 파일들은 무엇들이 있나요?

**Reply by x4cf** [reply]
커스텀으로 쉐이더를 작성해서 사용 가능합니다.
작동하지 않는 함수를 응용하는 스크립트와 연동된 쉐이더나, 사용불가능한 C#코드로 이루어진 에셋과 연동된경우 등 일부는 사용 불가능합니다

**Reply by smilejsu82** [reply]
만들면서 문제가 생기면 문의 해가면서 진행해야 겠네욧 
답변 감사합니다
